src/main/java/io/anserini/search/SearchArgs.java

```diff
@@ -84,6 +84,15 @@
   @Option(name = "-axiom", usage = "use Axiomatic query expansion model for the reranking")
   public boolean axiom = false;
 
+  @Option(name = "-axiom.decisive", usage = "decisive the expansion terms axiomatic reranking")
```

Comment:
`-axiom.deterministic` sounds more accurate?

Reply:
Also, you don't want the seed to be settable in the cmd line?

Reply:
Well, I just feel nobody is going to use it unless for deterministic results.... 
But I can add it anyway.